### PR TITLE
feat: [LM-1565] make checkboxfield focusable

### PR DESCRIPTION
**Pull request by lapapooh**
**PR description:**

When you disable the ripple effect in Material-UI components, it also disables the focus behavior because the ripple effect is tightly coupled with the focus/interaction states. This is especially true in older versions of Material-UI (v4 and earlier).

I think there are 3 ways of making checkboxfield focusable in material v4:
1. remove disableRipple={true}
2. use CSS to hide the ripple effect while keeping the prop disabled
3. upgrade to v5 :)

I chose the 1st option, let me know if you disagree
 

**Implemented:**

**Checklist:**

- [x] I ran this code locally
- [ ] I wrote the necessary tests
- [x] My code follows the [style guidelines](http://bit.ly/sd-web-style-guide)
- [x] I followed the [instructions](http://bit.ly/sd-web-pr) to create a pull request

**JIRA card:**
https://superdispatch.atlassian.net/browse/LM-1565

**Should know about:**

### CODE REVIEW

**Review by umrzoq-toshkentov — APPROVED** [review]

File: packages/testutils/package-lock.json
Comment (doniyor2109) [inline packages/testutils/package-lock.json:1]: Lets remove lock files

File: packages/ui/src/checkbox/CheckboxField.tsx [hunk 1]
```diff
@@ -58,14 +58,7 @@ export const CheckboxField: ForwardRefExoticComponent<CheckboxFieldProps> =
           checked={checked}
           onBlur={onBlur as FormControlLabelProps['onBlur']}
           onChange={onChange as FormControlLabelProps['onChange']}
-          control={
-            <Checkbox
-              ref={ref}
-              color="primary"
-              disableRipple={true}
```
[the remark this answers is not in the document]
Comment (doniyor2109) [reply packages/ui/src/checkbox/CheckboxField.tsx:65]: If you can get insights from git history, that would be great

[the remark this answers is not in the document]
Comment (lapapooh) [reply packages/ui/src/checkbox/CheckboxField.tsx:65]: @doniyor2109 according to the file history this prop was present from the very start

[the remark this answers is not in the document]
Comment (lapapooh) [reply packages/ui/src/checkbox/CheckboxField.tsx:65]: https://github.com/superdispatch/web-ui/pull/71

[the remark this answers is not in the document]
Comment (doniyor2109) [reply packages/ui/src/checkbox/CheckboxField.tsx:65]: Thanks for looking into it

**Review by doniyor2109 — APPROVED** [review]

**Pull request merged**